### ISSUE TITLE
[Feature Request] Title Here

### ISSUE THREAD

**Issue by CTP0805** [jagrosh/MusicBot#946]
### Is your feature request related to a problem? Please describe.

For the display of English playlist, change the button on the next page to instructions instead of emojis.

### What is your ideal solution to the problem?

For example: Grovy's previous page switch

### How would this feature be used?

By text command

### Checklist

- [X] I have checked [the documentation](https://jmusicbot.com/) to make sure this feature doesn't already exist
- [X] I have searched for [similar feature requests](https://github.com/jagrosh/MusicBot/labels/enhancement)
- [X] I have confirmed that my requests is **not** on the [list of things that will not be added](https://jmusicbot.com/things-that-wont-be-added)
- [X] I am running the latest version of the bot: ![Release](https://img.shields.io/github/release/jagrosh/MusicBot.svg)

**Comment by CTP0805** [comment]
Also, I hope that the list of music I played can be kept in the list after the song is played, and displayed by number

**Comment by MichailiK** [comment]
I unfortunately can't decipher what you are trying to say. Could you rephrase your request?

**Comment by Sanduhr32** [comment]
> Also, I hope that the list of music I played can be kept in the list after the song is played, and displayed by number

Basically a history list / bag allowing you to find/play played music

**Comment by Sanduhr32** [comment]
> ### Is your feature request related to a problem? Please describe.
> 
> For the display of English playlist, change the button on the next page to instructions instead of emojis.
> 
> 
> ### What is your ideal solution to the problem?
> 
> 
> For example: Grovy's previous page switch
> 
> ### How would this feature be used?
> 
> By text command

The user requested to have text labeling button menus instead of emojis.

**Comment by MichailiK** [comment]
Oh, button support. That's going to take a while.
